### PR TITLE
mention RNTester app in contributor guide

### PR DESCRIPTION
**Pull request by vonovak**
## Summary

motivation is following - the RNTester app is imho the best place to try out any changes that a contributor would make, yet it is not directly mentioned in the contributor guide. This fixes it.

## Changelog

[Internal] - Docs

## Test Plan

not necessary

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -72,18 +72,21 @@ We recommend referring to the [CONTRIBUTING](https://github.com/facebook/react-n
...
-2. If you've added code that should be tested, add tests.
-3. If you've changed APIs, update the documentation.
-4. Ensure the test suite passes, either locally or on CI once you opened a pull request.
-5. Make sure your code lints (for example via `yarn lint --fix`).
-6. Push the changes to your fork.
-7. Create a pull request to the React Native repository.
-8. Review and address comments on your pull request.
+2. Make the desired changes to React Native sources. Use the RNTester app to test them out.
+3. If you've added code that should be tested, add tests.
+4. If you've changed APIs, update the documentation, which lives in [another repo](https://github.com/facebook/react-native-website/pull/1469).
```
Comment (hramos) [inline CONTRIBUTING.md]: I think this should be a link to the repo root, and not a specific PR

Comment (vonovak) [reply CONTRIBUTING.md]: oh snap, that's right :D fixed it

**Comment by vonovak** [comment]
@hramos could you please take a look at this? it should be ready to merge. Thanks.

**Comment by vonovak** [comment]
@hramos seems like the import got stuck. I just rebased, can you please try again? thanks!

**Pull request closed without merging**